src/common/config_opts.h

```diff
@@ -1360,6 +1360,7 @@ OPTION(rgw_enable_usage_log, OPT_BOOL, false) // enable logging bandwidth usage
 OPTION(rgw_ops_log_rados, OPT_BOOL, true) // whether ops log should go to rados
 OPTION(rgw_ops_log_socket_path, OPT_STR, "") // path to unix domain socket where ops log can go
 OPTION(rgw_ops_log_data_backlog, OPT_INT, 5 << 20) // max data backlog for ops log
+OPTION(rgw_socket_backlog, OPT_INT, 1024) // socket backlog for fastcgi
```

Comment:
can we name this `rgw_fcgi_socket_backlog` so it's easier to tell that it only applies to fcgi?